compiler/src/dotty/tools/dotc/typer/ProtoTypes.scala

```diff
@@ -632,7 +632,7 @@ object ProtoTypes {
         normalize(et.resultType, pt)
```

Comment:
Just to be sure, adding the `followIFT` argument will only affect the fall through case and will be dropped for recursive calls. Is this intended?

Reply:
Yes, because all recursive normalize calls are against the same `pt`, which determined the value of `followIFT`.

Reply:
I just wondered whether the check can be inlined at 635 to 
```scala
if iftp.exists && !defn.isContextFunctionType(pt) then ...
```
to avoid this additional argument. But this would not be a refactoring since the argument `followIFT` defaults to `true` for the next recursive call.

Reply:
yes, exactly.